tools/bundle-size-tools/src/ADO/Constants.ts

```diff
@@ -16,19 +16,19 @@ export interface IADOConstants {
   // The ID for the build that runs to validate PRs
   // Used to update tagged PRs on CI build completion
   // Note: Assumes CI and PR builds both run in the same org/project
-  prBuildDefinitionId: number | undefined,
+  prBuildDefinitionId?: number,
```

Comment:
Although the typing is pretty much the same, just FYI on the subtle difference here:
`?:` implies optional, meaning objects can satisfy this without having any `prBuildDefinitionId` property at all.
`| undefined` still needs to have the property, but its value can be undefined.

```typescript
{
    bundleAnalysisArtifactName: "test"
}
```
The above works after this change, but not before. Before it must be:
```typescript
{
    bundleAnalysisArtifactName: "test",
    prBuildDefinitionId: undefined,
}
```

Reply:
these correspond to optional features so i think it makes sense in this case